app/controllers/AuthorizedToViewController.scala

```diff
@@ -274,14 +282,14 @@ class AuthorizedToViewController @Inject()(authenticate: IdentifierAction,
     }.head
   }
 
-  private def getDisplayLink(searchedAuthorities: SearchedAuthorities) = {
+  private def getDisplayLink(searchedAuthorities: SearchedAuthorities): Boolean = {
     searchedAuthorities.authorities.exists {
       case AuthorisedDutyDefermentAccount(_, balances) => balances.map(_.periodAvailableAccountBalance).isEmpty
       case AuthorisedCashAccount(_, availableAccountBalance) => availableAccountBalance.isEmpty
       case AuthorisedGeneralGuaranteeAccount(_, availableGuaranteeBalance) => availableGuaranteeBalance.isEmpty
     }
   }
 
-  protected def stripWithWhitespace(str: String): String =
+  private def stripWithWhitespace(str: String): String =
     str.replaceAll("\\s", emptyString).toUpperCase
```

Comment:
does this need to be on a separate line? I don't think it has length issues

Reply:
Done, made one single line